compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1676,17 +1676,41 @@ TR::Register *OMR::Power::TreeEvaluator::vimaxEvaluator(TR::Node *node, TR::Code
 
 TR::Register *OMR::Power::TreeEvaluator::vandEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::inlineVectorBinaryOp(node, cg, TR::InstOpCode::vand);
+   switch (node->getDataType())
+      {
+      case TR::VectorInt8:
+      case TR::VectorInt16:
+      case TR::VectorInt32:
+         return TR::TreeEvaluator::inlineVectorBinaryOp(node, cg, TR::InstOpCode::vand);
+      default:
+         return TR::TreeEvaluator::inlineVectorBinaryOp(node, cg, TR::InstOpCode::xxland);
```

Comment:
I think my comments around these functions were all lost, likely when a new commit was pushed.

My comment was to only have a single function call since the cases only differ by the opcode.  The switch should be to choose a different opcode.  You will generate far less code with only a single call.  I suggest that these case statements (and the ones below) all be restructured this way.